test/worlds/quadcopter.sdf

```diff
@@ -91,7 +91,7 @@
           </geometry>
         </collision>
         <visual name="rotor_0_visual">
-          <pose>0 0 0 1.57 0 0 0</pose>
+          <pose>0 0 0 1.57 0 0</pose>
```

Comment:
I realized last week while trying to fix another test that the quadcopter test is passing even though this SDF was invalid. I'll be making the same changes in other branches.